android_mobile/src/main/java/com/alexstyl/specialdates/events/namedays/NamedayLocale.java

```diff
@@ -11,6 +11,7 @@
     ROMANIAN("ro", false, R.string.Romanian, R.raw.ro_namedays),
     RUSSIAN("ru", false, R.string.Russian, R.raw.ru_namedays),
     LATVIAN("lv", false, R.string.Latvian, R.raw.lv_namedays),
+    LATVIANEXTENDED("lvext", false, R.string.LatvianExt, R.raw.lv_ext_namedays),
```

Comment:
I had to change the country code to "lv_LV" since the radio list cannot have same key. Another option would be to have only one "Latvian" for selection and if selected a checkbox for "Extended" option to appear. I took the name "lv_LV" from the link that you posted above :)